insights/specs/insights_archive.py

```diff
@@ -140,6 +140,7 @@ class InsightsArchiveSpecs(Specs):
     oc_get_rolebinding = simple_file("insights_commands/oc_get_rolebinding_-o_yaml_--all-namespaces")
     oc_get_route = simple_file("insights_commands/oc_get_route_-o_yaml_--all-namespaces")
     oc_get_service = simple_file("insights_commands/oc_get_service_-o_yaml_--all-namespaces")
+    oc_logs_dc_router = simple_file("insights_commands/oc_logs_dc/router_--timestamps=true_-n_default")
```

Comment:
Since the `simple_command` of this spec will add a sub-dir under the `insights_commands`, we need do more test to ensure this `simple_file` works well for this parser.